taxi/tests/test_models.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Driver, Car
+
+MANUFACTURER = Manufacturer(
+    name="TestName",
+    country="TestCountry",
+)
+DRIVER = Driver(
+    username="TestUsername",
+    password="password",
+    first_name="TestFirstName",
+    last_name="TestLastName",
+)
+
+CAR = Car(
+    model="TestModel",
+    manufacturer=MANUFACTURER,
+)
```

Comment:
Consider moving the creation of `MANUFACTURER`, `DRIVER`, and `CAR` instances into the test methods or a setup method. Using global variables for test data can lead to shared state between tests, which is not ideal for test isolation.